### PR TITLE
MetaMaint Chems Nerf or Nothing

### PR DESCRIPTION
**Pull request by Zytolg**
I've been asked by @EOBGames to downgrade the chemistry setup in Metastation's Abandoned Medbay.

## Changelog
:cl:
qol: There's some eye candy in the Abandoned Medbay that now suggests that it was at some point a medbay. 
balance: Downgrades the Chem Dispenser and Heater to their Ghetto Counterparts
/:cl:

### CODE REVIEW

File: _maps/map_files/MetaStation/MetaStation.dmm [hunk 1]
```diff
@@ -12010,13 +12019,12 @@
...
-/turf/open/floor/plating,
+/turf/open/floor/iron/showroomfloor,
 /area/maintenance/starboard/secondary)
 "cmb" = (
-/obj/machinery/chem_master{
-	pixel_x = -4
-	},
-/turf/open/floor/plating,
+/obj/structure/frame/machine,
+/obj/item/circuitboard/machine/chem_dispenser,
```
Comment (Capsandi) [inline _maps/map_files/MetaStation/MetaStation.dmm:12026]:
```suggestion
/obj/item/circuitboard/machine/chem_master,
```
Did you mean to put a chem-master board in here(as to allow separation of chemicals)? A whole chem-dispenser makes the setup rival the pharmacy, and removes most of the ghetto in ghetto-chem.

Comment (Zytolg) [reply _maps/map_files/MetaStation/MetaStation.dmm:12026]: Thanks for the catch, I'll make this change right away.

**Comment by TheBonded** [comment]
While you're at it, as far as I know the peacekeeper plushie is still lacking a name

**Comment by Zytolg** [comment]
This is currently waiting on #59580 to fix the bad overlays with Ghetto-ized Heaters.

**Comment by EOBGames** [comment]
Applying the red label for now, until #59580 is merged

**Pull request merged**